apps/service/.env.example

```
@@ -2,13 +2,24 @@ PORT=3005
 NODE_ENV=development
 ENV=development
 
+THROTTLE_LIMIT=10
+THROTTLE_TTL=1000
+RATELIMIT_WINDOW_MS=60000
+RATELIMIT_MAX=10000
+CORS_ALLOWED_ORIGIN=http://localhost:3002,http://localhost:3001,http://localhost:3000,http://127.0.0.1:3001
+IP_DENY_LIST=
+IP_ALLOW_LIST=
+MAINTENANCE_MODE=0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAINTENANCE_MODE key should go before the RATELIMIT_MAX key

(UnorderedKey)